tests/Unit/UtilizationMetricsApisTest.php

```diff
@@ -0,0 +1,436 @@
+    AiThread::factory()->count($randomRecords)->saved()->create();
+
+    $softDeleteAiThread = AiThread::factory()->saved()->create();
+    $softDeleteAiThread->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['saved_ai_chats'])->toBe($randomRecords);
+});
+
+it('checks the API returns Saved Prompts', function () {
+    $randomRecords = random_int(1, 10);
+
+    Prompt::factory()->count($randomRecords)->create();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['saved_prompts'])->toBe($randomRecords);
+});
+
+it('checks the API returns Prompts Inserted', function () {
+    $randomRecords = random_int(1, 10);
+
+    PromptUse::factory()->count($randomRecords)->create();
+
+    $softDeletePromptUse = PromptUse::factory()->create();
+    $softDeletePromptUse->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['prompts_inserted'])->toBe($randomRecords);
+});
+
+it('checks the API returns Retention CRM Users', function () {
+    $randomRecords = random_int(1, 10);
+
+    $licenseSettings = app(LicenseSettings::class);
+
+    $licenseSettings->data->limits->retentionCrmSeats = $randomRecords + 1;
+    $licenseSettings->save();
+
+    User::factory()->count($randomRecords)->licensed([LicenseType::RetentionCrm])->create();
+
+    $softDeleteRetentionCrmUser = User::factory()->licensed([LicenseType::RetentionCrm])->create();
+    $softDeleteRetentionCrmUser->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['retention_crm_users'])->toBe($randomRecords);
+});
+
+it('checks the API returns Recruitment CRM Users', function () {
+    $randomRecords = random_int(1, 10);
+
+    $licenseSettings = app(LicenseSettings::class);
+
+    $licenseSettings->data->limits->recruitmentCrmSeats = $randomRecords + 1;
+    $licenseSettings->save();
+
+    User::factory()->count($randomRecords)->licensed([LicenseType::RecruitmentCrm])->create();
+
+    $softDeleteRecruitmentCrmUser = User::factory()->licensed([LicenseType::RecruitmentCrm])->create();
+    $softDeleteRecruitmentCrmUser->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['recruitment_crm_users'])->toBe($randomRecords);
+});
+
+it('checks the API returns Student Records', function () {
+    $randomRecords = random_int(1, 10);
+
+    $totalRecords = $randomRecords + (Student::count());
+
+    Student::factory()->count($randomRecords)->create();
+
+    $softDeleteStudent = Student::factory()->create();
+    $softDeleteStudent->delete();
+
+    $response = get('/api/utilization-metrics');
```

Comment:
We should be using the route name here, not calling the path directly.

Reply:
Done